Jenkinsfile

```
@@ -245,6 +245,8 @@ void setupDomain() {
     echo '*#*#*#*#*#*#*#*#*#*#*#*#  Setting up tests  *#*#*#*#*#*#*#*#*#*#*#*#*#*#*#'
     makeDomain()
     sh "${ASADMIN} start-domain ${DOMAIN_NAME}"
+    sh "${ASADMIN} delete-jvm-options -Xmx512m"
+    sh "${ASADMIN} create-jvm-options -Xmx2g"
```

```suggestion
    sh "${ASADMIN} create-jvm-options -Xmx2g"
    sh "${ASADMIN} delete-jvm-options --target default-config -Xmx512m"
    sh "${ASADMIN} create-jvm-options --target default-config -Xmx2g"
```
Just in case - quicklook makes additional instances